datadog_lambda/dsm.py

```diff
@@ -1,38 +1,79 @@
+import json
+
+from ddtrace.internal.logger import get_logger
```

Comment:
This isn't in ddtrace so we shouldn't use the logger from it. Find a spot where we use a logger somewhere else in this repo and follow that.